src/visualization/Viewer.js

```diff
@@ -25,7 +25,7 @@ ROS3D.Viewer = function(options) {
   var width = options.width;
   var height = options.height;
   var background = options.background || '#111111';
-  var antialias = options.antialias;
+  var antialias = options.antialias || true;
```

Comment:
This can never be set to false if it defaults to true. If someone passed in `{antialias: false}` we would have `var antialias = false || true;` which equals true.